src/asserts/date-assert.js

```diff
@@ -4,19 +4,40 @@
  */
 
 import { Violation } from 'validator.js';
+import { isString } from 'lodash';
 
 /**
  * Export `DateAssert`.
  */
 
-export default function dateAssert() {
+export default function dateAssert({ format } = {}) {
+  /**
+   * Optional peer dependencies.
+   */
+
+  const moment = require('moment');
```

Comment:
Also, this is missing from package.json as a dev dependency otherwise tests will fail.